lib/src/components/NxCollapsibleItems/NxCollapsibleItems.scss

```diff
@@ -74,6 +74,10 @@ $-form-child-left-padding: calc(#{$-trigger-side-padding} + #{$-triangle-box-wid
   }
 }
 
+.nx-collapsible-items__tooltip-wrapper {
+  display: contents;
```

Comment:
This breaks the tooltip placement unfortunately

Reply:
> This breaks the tooltip placement unfortunately

I think the better solution is to add the `.nx-collapsible-items__header` styles to this element as well.